ee/fe/modules/340-monitoring-applications/templates/base-service-monitor.yaml

```diff
@@ -22,27 +22,27 @@
   replacement: $1
   targetLabel: endpoint
 
-# HTTP Scheme
+  # HTTP Scheme
```

Comment:
Why we move this one step right ? I think it is Goland's work.

Reply:
This one I moved this one intentionally to be in line with the condition.